.github/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 <h3>Improvements</h3>
 
+* `qnn.KerasLayer` now supports serialization when used in Keras "Functional"
+  API. [(#953)](https://github.com/PennyLaneAI/pennylane/pull/953)
+  
+  This unlocks additional functionality from `tf.keras`, e.g. cloning 
+  models:
```

Comment:
Suggest to focus this more on model cloning than serialization.

Reply:
Dito.